packages/story-editor/src/app/story/actions/useSaveStory.js

```diff
@@ -68,7 +71,12 @@ function useSaveStory({ storyId, pages, story, updateStory }) {
 
       return saveStoryById({
         storyId,
-        ...getStoryPropsToSave({ story, pages, metadata }),
+        ...getStoryPropsToSave({
+          story,
+          pages,
+          metadata,
+          args: { enableBetterCaptions },
```

Comment:
Nit: normally would discourage prop bags but with PropTypes it's better.

Reply:
A possible alternative might be to use `<FlagsProvider>` within the individual Output components